cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js

```diff
@@ -147,6 +150,12 @@ class HydratorPlusPlusTopPanelCtrl {
       artifact: this.HydratorPlusPlusConfigStore.getArtifact()
     };
   }
+  setActiveNodes() {
+    this.hasNodes = !!this.DAGPlusPlusNodesStore.getNodes().length;
```

Comment:
this can be just `this.hasNodes = this.DAGPlusPlusNodesStore.getNodes().length;` ? `0` is anyways a falsy value right.